File: core-java-practice/gcr-codebase/AreaOfCircle.java
import java.util.*;

public class AreaOfCircle {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // Input the radius of the circle
        System.out.print("Enter the radius of the circle: ");
        int radius = sc.nextInt();

        int area= (int) (Math.PI * radius * radius);

        // Output result
        System.out.println("Area of the cirle: " + area);

        
    }
}

File: core-csharp-practice/gcr-codebase/AreaOfCircle.cs

class AreaOfCircle
{
    static void Main(string[] args)
    {
        // Input the radius of the circle
        Console.Write("Enter the radius of the circle: ");
        int radius = Convert.ToInt32(Console.ReadLine());

        // Calculate area
        int area = (int)(Math.PI * radius * radius);

        // Output result
        Console.WriteLine("Area of the circle: " + area);
    }
}
